gradle/tests/com.liferay.ide.gradle.core.tests/META-INF/MANIFEST.MF

```diff
@@ -10,6 +10,10 @@ Require-Bundle: org.eclipse.core.runtime,
  com.liferay.ide.gradle.core;bundle-version="3.0.0",
  org.eclipse.jdt.core,
  org.eclipse.ui.console,
- org.gradle.toolingapi
+ org.gradle.toolingapi,
+ org.eclipse.buildship.core,
+ org.eclipse.ui.ide,
```

Comment:
Is the org.eclipse.ui.ide project required?  does buildship.core require other *ui* plugins?  if so, then we need to make this a gradle.ui.tests folder instead and make it a ui tests not a core tests.